### PR TITLE
Bug fixes and aliases

### PR DESCRIPTION
**Pull request by ericcurtin**
It should be self.pull(args).

Also added 'ls' alias for 'list'.

### CODE REVIEW

**Comment by ericcurtin** [comment]
The ls alias is because ollama has the "rollama ls" alias for "ollama list", it's also kinda ingrained into a lot of people's muscle memory to type ls.

File: docs/ramalama-ls.1.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+% ramalama-ls 1
```
Comment (rhatdan) [inline docs/ramalama-ls.1.md:1]: We should just use man page aliasing.

cat podman/docs/source/markdown/links/podman-container-ls.1 
.so man1/podman-ps.1

cat ramalama-ls.1
.so man1/ramalama-ls.1

Comment (rhatdan) [reply docs/ramalama-ls.1.md:1]: Than add ramalama-ls to ramalama-list.md man page info

Comment (rhatdan) [reply docs/ramalama-ls.1.md:1]: Let's not duplicate the number of man pages, but link them together.

Everything else in this PR is great.

File: hack/man-page-checker [hunk 2]
```diff
@@ -19,7 +19,11 @@ cd $(dirname $0)/../docs/ || die "Please run me from top-level libpod dir"
 
 rc=0
 
-for md in *.1.md;do
+for md in *.1.md; do
```
Comment (ericcurtin) [inline hack/man-page-checker:22]: The podman version has a more complex way of skipping aliases, it patches a perl script, etc. Would rather use this way for now

File: docs/ramalama-ls.1.md (hunk 1 above)
Comment (ericcurtin) [reply docs/ramalama-ls.1.md]: Done

File: hack/man-page-checker [hunk 3]
```diff
@@ -19,7 +19,11 @@ cd $(dirname $0)/../docs/ || die "Please run me from top-level libpod dir"
...
+    if [ "$md" = "ramalama-ls.1.md" ]; then
```
Comment (rhatdan) [inline hack/man-page-checker:23]: If the link is not an md file then this is not necessary.

File: docs/ramalama-ls.1.md [hunk 4]
```diff
@@ -0,0 +1 @@
+.so man1/ramalama-list.1
```
[the remark this answers is not in the document]
Comment (ericcurtin) [reply docs/ramalama-ls.1.md]: Yup that works, thanks for spotting, re-pushed with changes.

File: hack/man-page-checker (hunk 3 above)
Comment (ericcurtin) [reply hack/man-page-checker]: Yup that works, thanks for spotting, re-pushed with changes.

File: hack/xref-helpmsgs-manpages [hunk 5]
```diff
@@ -178,7 +178,7 @@ sub xref_by_help {
 
   OPTION:
     for my $k (sort keys %$help) {
-        next if $k =~ /^_/;             # metadata ("_desc"). Ignore.
```
Comment (ericcurtin) [inline hack/xref-helpmsgs-manpages:181]: We still need this skip now, but only this one.

File: ramalama/oci.py [hunk 6]
```diff
@@ -20,7 +20,7 @@ def login(self, args):
             conman_args.extend(["--password", args.password])
         if args.passwordstdin:
             conman_args.append("--password-stdin")
-        conman_args.append(self.transport)
+        conman_args.append(args.transport)
```
Comment (ericcurtin) [inline ramalama/oci.py:23]: Added another small bugfix here

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**